src/test/lib/logical_query_plan/change_meta_table_node_test.cpp

```diff
@@ -44,4 +44,8 @@ TEST_F(ChangeMetaTableNodeTest, Copy) {
   EXPECT_EQ(*_change_meta_table_node, *_change_meta_table_node->deep_copy());
 }
 
+TEST_F(ChangeMetaTableNodeTest, NoUniqueColumnCombinations) {
+  EXPECT_THROW(_change_meta_table_node->unique_column_combinations(), std::logic_error);
```

Comment:
Comment why that fails? Looks like a valid call.

Reply:
Gonna add that. Non-query nodes should not be asked for dependencies.